### DISCUSSION TITLE
Geräte-IDs aus HM/Sunnyportal auslesen

### DISCUSSION THREAD

**Discussion by seb2fis** [camueller/SmartApplianceEnabler#624] [Q&A]
Hallo zusammen!

Nachdem mir mein Raspi abgeraucht ist, musst ich auf einem anderen Gerät neu installieren... allerdings weiß ich nicht mehr, welche ID ich für welches Gerät vergeben hatte! Ich hatte mich auf jeden Fall nach der Empfehlung in der Anleitung gerichtet und mein Geburtsdatum für den vorderen Teil verwendet und dann von 1 an der letzten Stelle nach oben gezählt.
Hat jemand eine Ahnung, wie ich das jetzt wieder auslesen kann, damit ich weiß, welche ID ich welchem Gerät geben muss damit die Daten im Sunnyportal stimmen?
Vielen Dank im voraus für Eure Ideen!
LG
Sebastian

**Comment by camueller** [comment]
Mir ist kein Weg bekannt, wie Du wieder an die IDs kommst. Wenn Du auch keine Möglichkeit dazu siehst, wirst Du wohl die Geräte im SHM löschen müssen und neu anlegen. Die bisherigen Verbräuche sind dann natürlich weg.

Und beim nächsten Mal ein Backup der Konfig-Files anlegen, der Hinweis steht extra in der Doku ...

**Reply by seb2fis** [reply]
Vielen Dank für die schnelle Antwort, dann muss ich wohl in den sauren Apfel beißen. Kann Backup, kein Mitleid ;)
